tendrl/node_agent/discovery/platform/manager.py

```diff
@@ -30,8 +30,16 @@ def load_plugins(self):
                 for name, cls in clsmembers:
                     exec("from %s import %s" % (plugin_name, name))
         except (SyntaxError, ValueError, ImportError) as ex:
-            LOG.error('Failed to load the platform plugins. Error %s' %
-                      ex, exc_info=True)
+            Event(
+                Message(
+                    priority="error",
+                    publisher=tendrl_ns.publisher_id,
+                    payload={"message": 'Failed to load the platform plugins. '
+                                        'Error %s %s' %
+                                        (ex, traceback.format_exc())
+                             }
+                )
+            )
```

Comment:
For this you can use exception message class but it is not merged if it is merged then change as per that